pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md

```diff
@@ -0,0 +1,92 @@
+
+## Namespaces
+
+| Predicate Prefix | Rdf-vocab Predicate Prefix | Namespace |
+| -------- | --------- | -------- |
+| dce: | DC11: | http://purl.org/dc/elements/1.1/ |
+| dct: | DC: | http://purl.org/dc/terms/ |
+| edm: | EDM: | http://www.europeana.eu/schemas/edm/ |
+| foaf: | FOAF: | http://xmlns.com/foaf/0.1/ |
+| rdfs: | RDFS: | http://www.w3.org/2000/01/rdf-schema# |
+| xsd: | | http://www.w3.org/2001/XMLSchema# |
+| mrel: | | http://id.loc.gov/vocabulary/relators/ |
+| schema: | SCHEMA: | https://schema.org/ |
+
+Distinction: The [MUST out of the box](https://github.com/samvera/hyrax/blob/master/app/forms/hyrax/forms/work_form.rb#L33) is a slightly different set of things from what is in Core.
+
+Reference: Local controlled vocabularies and forms pulled in here: [https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/config_generator.rb](https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/config_generator.rb)
+
+## Basic Metadata
+
+Basic metadata properties are defined in [app/models/concerns/hyrax/basic_metadata.rb](https://github.com/samvera/hyrax/blob/master/app/models/concerns/hyrax/basic_metadata.rb)
+
+[Expanded documentation for Hyrax Basic Metadata](https://docs.google.com/spreadsheets/d/1yZZvoQG6lANyqinMlxuOWT4W4ZIIyBJVrSii5laxEm4/edit?usp=sharing)
+
+New fields for Hyrax 3 are indicated with _italics_. Fields are listed in the order they appear in the Hyrax Work form by default.
+
+| Property (Field) | Predicate | Rdf-vocab Predicate | Recommendation | Expected Value (Data Type) |
+| ---------------- | --------- | -------- | -------- | -------- | 
+| creator          | dce:creator | ::RDF::Vocab::DC11.creator | MUST (Required) | xsd:string (Literal) |
+| rights_statement | edm:rights | ::RDF::Vocab::EDM.rights | MUST (Required) | xsd:anyUri |
+| alternative_title | dct:alternative | ::RDF::Vocab::DC.alternative | MAY | xsd:string (Literal) |
+| contributor      | dce:contributor | ::RDF::Vocab::DC11.contributor | MAY | xsd:string (Literal) |
+| description      | dce:description | ::RDF::Vocab::DC11.description | MAY | xsd:string (Literal) |
+| _abstract_         | _dct:abstract_ | _::RDF::Vocab::DC.abstract_ | _MAY_ | _xsd:string (Literal)_ |
+| keyword          | schema:keywords | ::RDF::Vocab::SCHEMA.keywords | MAY | xsd:string (Literal) |
+| license          | dct:license | ::RDF::Vocab::DC.license | MAY | xsd:anyURI |
+| _access_right_     | _dct:accessRights_ | _::RDF::Vocab::DC.accessRights_ | _MAY_ | _xsd:string (Literal)_ |
+| _rights_notes_     | _dct:rights_ | _::RDF::Vocab::DC.rights_ | _MAY_ | _xsd:string (Literal)_ |
```

Comment:
The DC Elements Rights is used in the definition, so these should be `dce:rights` and `::RDF::Vocab::DC11.rights`

Reply:
And made this change!